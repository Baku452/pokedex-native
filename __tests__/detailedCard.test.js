import React from 'react';
import renderer from 'react-test-renderer';
import DetailedCard from '../src/components/detailedCard/detailedCard.component';

test('renders correctly', () => {
  const tree = renderer.create(<DetailedCard />).toJSON();
  expect(tree).toMatchSnapshot();
});
