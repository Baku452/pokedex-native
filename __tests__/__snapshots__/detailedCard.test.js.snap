// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ActivityIndicator
  size="large"
/>
`;
